content/en/getting-started/auth-token.md

```diff
@@ -2,28 +2,37 @@
+
+The Auth Token remains unchanged unless manually rotated by the user, regardless of any license assignment changes. You can locate your Auth Token on the [Getting Started page](https://app.localstack.cloud/getting-started) or the [Auth Token page](https://app.localstack.cloud/workspace/auth-token) in the LocalStack Web Application.
+
+{{< alert title="Important" color="danger" >}}
+-   Previously, API keys were required to activate the LocalStack core cloud emulator. These API keys are now being replaced by Auth Tokens.
+-   Currently, LocalStack supports both API Keys and Auth Tokens. However, API Keys will be discontinued in the upcoming major release of LocalStack.
+-   To update your LocalStack configuration, replace your API Key with an Auth Token. Use the `LOCALSTACK_AUTH_TOKEN` environment variable in place of `LOCALSTACK_API_KEY`.
+{{< /alert >}}
 
 ## Managing your License
 
-To get started, you need to have a LocalStack license. If you don't have one, you can [sign up for a free trial](https://localstack.cloud/pricing/) without any credit card required. The free trial will last 14 days, and you can use it to test all the features of LocalStack.
+To begin using LocalStack, a license is required. If you don't already have a license, you can opt for a free 14-day trial by [signing up here](https://localstack.cloud/pricing/), with no credit card required. This trial allows full access to all LocalStack features.
 
-You can assign licenses to users on the [Users & Licenses](https://app.localstack.cloud/workspace/members) page or you can check your assigned license on the [My License](https://app.localstack.cloud/workspace/my-license) page.
+For license management, assign licenses to users via the [Users & Licenses page](https://app.localstack.cloud/workspace/members). To view your own assigned license, visit the [My License page](https://app.localstack.cloud/workspace/my-license).
 
 {{< alert title="Important" color="danger" >}}
-- Avoid sharing your auth token with anyone. Ensure that you do not commit it to any source code management systems (like Git repositories).
-- If you push an auth token to a public repository, it has potentially been exposed and might remain in the history (even if you try to rewrite it).
-- If you accidentally publish your auth token, you can rotate it on the [Auth Token](https://app.localstack.cloud/workspace/auth-token) page
-- Use in CI or other machine environments requires a CI key. Check out our [CI documentation]({{< ref "user-guide/ci" >}}) to see the proper way to handle secrets in your CI environment to store your CI key securely.
+-   It's crucial to keep your Auth Token confidential. Do not include it in source code management systems, such as Git repositories.
```

Comment:
note to self - we should get auth tokens included into GH Secrets scanning soon.. 👍